R/mod_metric_details_server.R

```diff
@@ -97,8 +98,24 @@ mod_metric_details_server <- function(
               HTML()
           })
           outputOptions(output, "results", suspendWhenHidden = FALSE)
+
+
         }
       )
     })
+
+
+    ### Highlight analysis output table and update site selector
+
+
+    observeEvent(rctv_strSite(), {
```

Comment:
I tried to find a way to make this behave properly upon returning to the tab, but no luck so far, unfortunately. We'll let it go for this iteration, as discussed!